docs/collections.md

```diff
@@ -1,20 +1,24 @@
-# Collections
+# Collections and Builders
 
-Robo provides task collections as a means of making error detection and recovery easier. When Robo tasks are added to a collection, their execution is deferred until the `$collection->run()` method is called.  When using collections, a Robo script will go through three phases:
+Robo provides task collections as a means of making error detection and recovery easier. When Robo tasks are added to a collection, their execution is deferred until the `$collection->run()` method is called.  
 
-1. Determine which tasks will need to be run, and create a collection.
+The easiest way to manage a collection of tasks is to use a task builder. Task builders allow tasks to be created via chained methods.  All of the tasks created by the same builder are added to a collection; when the `run()` method is called, all of the tasks in the collection run. If one of the tasks fail, then the operation will be aborted; rollback tasks may also be defined to restore the system to its original condition.
```

Comment:
before this paragraph pleas provide an example of creating and using basic collection (with code)